stdlib/StdlibModules

```diff
@@ -18,68 +18,71 @@
 # This file lists all standard library modules.
 # It is used in particular to know what to expunge in toplevels.
```

Comment:
While we're here, I think it would be worth referencing exactly where this is used, and how, which is in three places:

1. `stdlib/Makefile`, where it lists all the modules in `stdlib.cma`
2. `Makefile`, where it is used to determine all the module names to expunge in `ocaml`
3. `ocamldoc/Makefile.docfiles`, where it is used to determine all the files to generate documentation for the standard library

Reply:
More documentation sounds good indeed.